packages/sanity/tsdoc.json

```diff
@@ -5,6 +5,11 @@
       "tagName": "@hidden",
       "syntaxKind": "block",
       "allowMultiple": true
+    },
+    {
+      "tagName": "@todo",
+      "syntaxKind": "block",
+      "allowMultiple": true
```

Comment:
Not sure if we need this particular block since, as far as I understand it, we only need to add this to tsdoc (have it collected) if we intend on rendering it in the front end. I think that adding it in the "supportForTags" is probably enough

Reply:
While trying something else I realised that this is overall breaking as it is (still doesn't recognise the todo tag 🤔 )

Reply:
> I think that adding it in the "supportForTags" is probably enough

Good catch! Updated